app/javascript/controllers/popover_controller.js

```diff
@@ -3,6 +3,13 @@ import { Controller } from "@hotwired/stimulus"
 export default class extends Controller {
   connect() {
     const popoverTriggerList = document.querySelectorAll('[data-bs-toggle="popover"]')
-    const popoverList = [...popoverTriggerList].map((popoverTriggerEl) => new bootstrap.Popover(popoverTriggerEl))
+    this.popoverList = [...popoverTriggerList].map((popoverTriggerEl) => new bootstrap.Popover(popoverTriggerEl))
+  }
+
+  disconnect() {
+    this.popoverList.forEach((popover) => {
```

Comment:
I had to do this for the tooltip controller (which I kept around in this PR). Tooltips for breakpoint fields and buttons would stick around forever since the DOM (and event handlers) often gets rebuilt as things change, usually by interacting with those fields.